src/components/header/index.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactComponent as ProfileIcon } from '../../assets/profile.svg';
+import { ReactComponent as RingIcon } from '../../assets/ring.svg';
+import { ReactComponent as SearchIcon } from '../../assets/search.svg';
+import { fonts } from '../../theme';
+import { links } from '../navBar/helper';
+import { useLocation } from 'react-router-dom';
+import {
+  HeaderContainer,
+  IconsContainer,
+  PhotoContainer,
+  ProfileContainer,
+  StyledHeaderText,
+  StyledHr,
+  StyledTypography,
+} from './styled';
+
+const Header: React.FC = () => {
+  const location = useLocation();
+  const currentLink = links.find((link) => link.to === location.pathname);
+  const headerText = currentLink ? currentLink.name : 'Overview';
```

Comment:
it seems like we have `name: 'Overview',` in links array as well

Reply:
fix: 'Overview' changed to 'links[0].name' - so whatever the first element is - it will be the default